assets/wizards/popups/index.js

```diff
@@ -337,18 +321,20 @@ class PopupsWizard extends Component {
 										};
 
 										return (
-											<Campaigns
-												{ ...popupManagementSharedProps }
-												archiveCampaignGroup={ archiveCampaignGroup }
-												campaignId={ campaignId }
-												createCampaignGroup={ createCampaignGroup }
-												deleteCampaignGroup={ deleteCampaignGroup }
-												duplicateCampaignGroup={ duplicateCampaignGroup }
-												renameCampaignGroup={ renameCampaignGroup }
-												prompts={ filterByCampaign( prompts, campaignId ) }
-												campaigns={ campaigns }
-												hasUnassigned={ filterByCampaign( prompts, 'unassigned' ).length }
-											/>
+											<CampaignsContext.Provider value={ prompts }>
```

Comment:
Note: the use of context here creates a sort of source-of-truth duplication, since prompts (though filtered) are also passed in a prop to the `Campaigns` component. But this can be resolved in a separate PR, just flagging here.

Reply:
Ah, good point. Since this is the only other place `prompts` is passed directly as a prop, I just went ahead and refactored this in 98c7497. Now every child of `popups/index.js` that uses `prompts` directly uses it from the context data.